guides/source/development_dependencies_install.md

```diff
@@ -376,3 +376,28 @@ command inside of the `activestorage` directory to install the dependencies:
 ```bash
 yarn install
 ```
+
+Extracting previews, tested in ActiveStorage's test suite requires third-party
+applications, `FFmpeg` for video and `muPDF` for PDFs. Without these
+applications installed, ActiveStorage tests will raise errors.
+
+On macOS you can run:
+
+```bash
+brew install ffmpeg
+brew install mupdf-tools
```

Comment:
I think this is missing `poppler` and `xquartz`. See the Brewfile.

Reply:
Thank you, fixed.